Dockerfile.tools

```diff
@@ -2,7 +2,7 @@ FROM registry.access.redhat.com/ubi8/ubi
 
 ENV HOME=/home/ci
 ENV GOROOT=/usr/local/go
-ENV GOVERSION=1.18.2
+ENV GOVERSION=1.20
```

Comment:
`1.20.5`?

Reply:
we can't have specify minor version under `go.mod`
I went with 1.20 for consistency

Reply:
Changed to `1.20.5` lets see